src/libs/fileDownload/FileUtils.ts

```diff
@@ -256,6 +256,23 @@ function validateImageForCorruption(file: FileObject): Promise<{width: number; h
     });
 }
 
+function verifyFileFormat({fileUri, formatSignatures}: {fileUri: string; formatSignatures: readonly string[]}) {
```

Comment:
Could we add a comment explaining what this function is for. It's not super obvious to me.

Reply:
ofc